The co-ordinates for drag/drop events originating in plugin windows is
incorrect, logged mozilla bug on it here:
https://bugzilla.mozilla.org/show_bug.cgi?id=809601

Index: /as/komodo-devel/mozilla/build/moz1700-ko8.9/mozilla/widget/gtk2/nsWindow.cpp
--- a/widget/gtk2/nsWindow.cpp	Mon Sep 24 23:04:32 2012 +0200
+++ b/widget/gtk2/nsWindow.cpp	Wed Nov 07 11:55:32 2012 -0800
@@ -5614,13 +5614,13 @@
     }
 
     LOGDRAG(("nsWindow drag-motion signal for %p\n", (void*)innerMostWindow));
 
     return nsDragService::GetInstance()->
         ScheduleMotionEvent(innerMostWindow, aDragContext,
-                            nsIntPoint(aX, aY), aTime);
+                            nsIntPoint(retx, rety), aTime);
 }
 
 static void
 drag_leave_event_cb(GtkWidget *aWidget,
                     GdkDragContext *aDragContext,
                     guint aTime,
@@ -5684,13 +5684,13 @@
     }
 
     LOGDRAG(("nsWindow drag-drop signal for %p\n", (void*)innerMostWindow));
 
     return nsDragService::GetInstance()->
         ScheduleDropEvent(innerMostWindow, aDragContext,
-                          nsIntPoint(aX, aY), aTime);
+                          nsIntPoint(retx, rety), aTime);
 }
 
 static void
 drag_data_received_event_cb(GtkWidget *aWidget,
                             GdkDragContext *aDragContext,
                             gint aX,
